examples/pytorch/mixhop/main.py

```diff
@@ -0,0 +1,309 @@
+#!/usr/bin/env python3
+# -*- coding:utf-8 -*-
```

Comment:
You don't need these two lines as DGL only supports Python 3.

Reply:
Done.